internal/lua/module.go

```diff
@@ -1435,14 +1480,21 @@ func NewModule(args map[string][]string, state *State, socket vxproto.IModuleSoc
 
 	luar.Register(state.L, "__imc", luar.Map{
 		// Functions
-		"get_token":          m.getIMCToken,
-		"get_info":           m.getIMCTokenInfo,
-		"is_exist":           m.isIMCTokenExist,
-		"make_token":         m.makeIMCToken,
-		"get_groups":         m.getIMCGroupIDs,
-		"get_modules":        m.getIMCModuleIDs,
-		"get_groups_by_mid":  m.getIMCGroupIDsByMID,
-		"get_modules_by_gid": m.getIMCModuleIDsByGID,
+		"get_token":                   m.getIMCToken,
+		"get_info":                    m.getIMCTokenInfo,
+		"is_exist":                    m.isIMCTokenExist,
+		"is_topic":                    m.isIMCTopicExist,
```

Comment:
Maybe better use full names? - "is_topic_exists"?, I think that having "is_exist" already is a bit misleading and better to be renamed to "is_token_exists", or at least new name can be introduced along with "deprecation" of old one.

Reply:
I changed this part to combine is_exist and is_topic api methods